client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent, useEffect, useCallback, useState } from 'react';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormSelect, { FormSelectProps } from 'calypso/components/forms/form-select';
```

Comment:
Nitpick, you can import the FormSelectProps and FunctionComponent types separately as a type like as is done below with AkismetProQuantityDropDownProps